### ISSUE TITLE
A11y_VSCode_ WelcomeScreen_ScreenReader: The role and name of the interactive element is not announced to screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#76671]
**Environment Details:**
VSCode Version : 1.35.1 (user setup)
OS Version : Win10

GitHubTags: #A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11y_VSCodeEditor; #MAS4.1.2

**Additional Details:**
MAS Violated : MAS 4.1.2
ScreenReader Versions : NVDA2019.1.1, JAWS 2019

**Repro Steps:**
1. Open Visual studio code editor.
2. Using "Alt+H" key navigate to the "Help" menu bar option.
3. Using "Down" arrow key navigate to the "Welcome item".
4. Start screen reader.
5. Navigate to “Tools and Languages” and “Settings and keybindings” interactive elements.
6. Listen if it's name and role of these elements is announced or not.

**Actual:**
When screen reader users navigate to the “Tools and Languages” and “Settings and keybindings” interactive elements the name and the role of these elements are not announced to the screen reader users

This issue has been observed in "Win 10 + JAWS" and "Win10 + NVDA".

**Expected:**
The name and role of the interactive elements should be announced to the screen reader users when they navigate to it.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
If the name and role of the interactive elements are not announced to the screen reader users then they will not understand how to interact with them. 

**MAS Reference:**
MAS 4.1.2 - [https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Attachment for Reference:**
[VSCode_ WelcomeScreen_ScreenReader Role and Name not announced.pptx](https://github.com/microsoft/vscode/files/3361502/VSCode_.WelcomeScreen_ScreenReader.Role.and.Name.not.announced.pptx)
